client/executor/Cargo.toml

```diff
@@ -37,13 +37,17 @@ assert_matches = "1.3.0"
 wabt = "0.9.2"
 hex-literal = "0.2.1"
 runtime-test = { package = "substrate-runtime-test", path = "runtime-test" }
+test-runtime = { package = "substrate-test-runtime", path = "../../test/utils/runtime" }
+runtime-interface = { package = "substrate-runtime-interface", path = "../../primitives/runtime-interface" }
 client-api = { package = "substrate-client-api", path = "../api" }
 substrate-offchain = { path = "../offchain/" }
 state_machine = { package = "substrate-state-machine", path = "../../primitives/state-machine"  }
 test-case = "0.3.3"
 
 [features]
-default = []
+default = [ "std" ]
+# This crate does not have `no_std` support, we just require this for tests
```

Comment:
Can you elaborate more why this is needed? I thought that tests run in `std` anyway? We don't seem to be using that feature anywhere in the code afaict.

Reply:
`#[runtime_interface]` requires a `std` feature. Otherwise the generated code expects wasm. I could probably at some point change the feature selection to `any(std, test)`.